iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/AbstractIidmSerDeTest.java

```diff
@@ -103,6 +103,16 @@ protected void allFormatsRoundTripFromVersionedXmlFromMinToCurrentVersionTest(St
                 .toArray(IidmVersion[]::new));
     }
 
+    /**
+     * Execute a round trip test on the test resource IIDM-XML file with a given file name for all IIDM versions
+     * equals or more recent than a given minimum IIDM version.
+     */
+    protected void allFormatsRoundTripFromVersionedXmlFromMinVersionTest(String file, IidmVersion minVersion) throws IOException {
```

Comment:
not sure here here ... 

why not re-using existing `allFormatsRoundTripFromVersionedXmlFromMinToCurrentVersionTest` used already in many places ?

Reply:
oh I see, thanks for the explanation